### PR TITLE
added an ansible passowrd to allow ansible to ssh to the hosts

### PR DESCRIPTION
**Pull request by BonjeBongo**

### CODE REVIEW

File: inventory/inventory [hunk 1]
```diff
@@ -12,5 +12,8 @@
 10.0.1.2
 10.0.1.3
 
+[centos7:vars]
+ansible_password=student
```
Comment (jwflory) [inline inventory/inventory]: Normally [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) is used to manage secrets like this. To make lab submissions easier though, it's okay to leave it public as-is. Maybe something to mention in the sec report?

Comment (BonjeBongo) [reply inventory/inventory]: I guess, but even then, the playbook appears frozen after I run it

image

been sitting like that for the past 30 minutes or so

Comment (jwflory) [reply inventory/inventory]: @BonjeBongo Weird. Try running with more verbosity to get better explanations of what is going on underneath: `ansible-playbook -vv playbooks/master.yml`

Comment (BonjeBongo) [reply inventory/inventory]: Did that, after the ok for 10.0.1.3 its just blank. And just sits there. Check the Discord

**Pull request merged**